A32NX/fr-FR.locPak

```diff
@@ -8,13 +8,13 @@
       "AIRCRAFT.UI_VARIATION": "Airbus",
       "COCKPIT.TOOLTIPS.AUTOPILOT_PANEL_HDG_TUNING_LEFT": "Adapter le cap souhaité sur la gauche (%((A:AUTOPILOT HEADING LOCK DIR:1,degrees))%!d!°)",
       "COCKPIT.TOOLTIPS.AUTOPILOT_PANEL_HDG_TUNING_RIGHT": "Adapter le cap souhaité sur la droite (%((A:AUTOPILOT HEADING LOCK DIR:1,degrees))%!d!°)",
-      "COCKPIT.TOOLTIPS.APU_AVAILABLE": "APU available",
+      "COCKPIT.TOOLTIPS.APU_AVAILABLE": "APU disponible",
       "A32NX.LOADING.TIPS.A320NEO_001": "Le FlyByWire A32NX est le fruit d'une collaboration entre des centaines de contributeurs individuels.",
-      "A32NX.LOADING.TIPS.A320NEO_002": "Le saviez-vous? L'A320 peut atterrir par vent de travers jusqu'à 38 nœuds et avec un vent arrière jusqu'à 15 nœuds!",
+      "A32NX.LOADING.TIPS.A320NEO_002": "Le saviez-vous ? L'A320 peut atterrir par vent de travers jusqu'à 38 nœuds et avec un vent arrière jusqu'à 15 nœuds !",
       "A32NX.LOADING.TIPS.A320NEO_003": "FlyByWire tire son nom des commandes de vol électriques introduites pour la première fois dans l'Airbus A320.",
-      "A32NX.LOADING.TIPS.A320NEO_004": "Il y a 9693 avions de la famille A320 construits! C'est presque autant que la population du Tuvalu!",
-      "A32NX.LOADING.TIPS.A320NEO_005": "Les avions Airbus A320neo ont une autonomie de 3500 NM ou 6500 KM. C'est suffisant pour voler de New York à Paris!",
-      "A32NX.LOADING.TIPS.A320NEO_006": "Le saviez-vous? La famille A320 a 4 variantes! L'A318, l'A319, l'A320 et l'A321."
+      "A32NX.LOADING.TIPS.A320NEO_004": "Il y a 9693 avions de la famille A320 construits ! C'est presque autant que la population des Tuvalu !",
+      "A32NX.LOADING.TIPS.A320NEO_005": "Les avions Airbus A320neo ont une autonomie de 3500 NM ou 6500 km. C'est suffisant pour voler de New York à Paris !",
```

Comment:
what about the nm? do you guys write km lower case but nm upper case?

Reply:
Good question ! I doubled check on [Wikipedia](https://fr.wikipedia.org/wiki/Mille_marin) but also this [paper](https://www.nist.gov/system/files/documents/pml/div684/fcdc/si_brochure_8.pdf), from International Bureau of Measurements (approximate translation) : check the page 42, table 8, (d). There is not international agreement on the unit, but it’s commonly written NM.

Reply:
I looked it up myself and the upper case NM is actually the agreement I found in US and Germany.

![image](https://user-images.githubusercontent.com/1857322/112721091-5002cf00-8f02-11eb-99c0-ba11187ad8e1.png)

Reply:
There is actually [guidance on this](https://aerosavvy.com/wp-content/uploads/2014/08/an05_cons.pdf) from ICAO: 

![image](https://user-images.githubusercontent.com/929769/112817283-8c5d3900-9082-11eb-95c3-b425375347af.png)

Specifically the SI unit km is lowercase, but the alternative NM is uppercase.